aten/src/ATen/native_parse.py

```diff
@@ -144,12 +179,14 @@ def run(paths):
                     func_decl, return_decl = [x.strip() for x in func['func'].split('->')]
                 else:
                     raise Exception('Expected return declaration')
-                fn_name, arguments = func_decl.split('(')
-                arguments = arguments.split(')')[0]
+                func_decl = func_decl.strip()
+                fn_name_start = func_decl.find("(")
+                fn_name = func_decl[:fn_name_start]
+                arguments = func_decl[fn_name_start + 1:-1]
```

Comment:
these aren't the arguments (at least how they were defined previously), and you already get the return_arguments later.  Can you clean this up?